submissions/omolayoOdupe/HTML/Product Page 138.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>BUSYBASKET PACKAGING</title>
+</head>
+<body>
+    <h2> BusyBasket Packaging</h2>
+    <p> It is the festive season, we are your best option for that gift items to your loved ones </p>
+    <nav> 
+        <table>
+            <tr>
+                <th><h4> Home </h4></th>
+                <th><h4> About Us </h4></th>
+                <th><h4> Contact Us</h4></th>
+                <th><h4> FAQ </h4></th>
+                <th><h4> Blog </h4></th>
+                <th><h4> Customer Feedback </h4></th>
+            </tr>
+        </table>
+    </nav> 
+    <section>
+        <h3> Product Feature </h3>
+            <h3><strong><em> Our Preferred Pack </em></strong></h3>
+                <p> We already have a well packed hampers for you to choose from with varying price range </p>
+                <img src="/Users/Odupe Olukayode/Desktop/Dufuna-Fem/submissions/omolayoOdupe/HTML/Hamper Basket.jpg"> Price: N10,000 - N45,000
```

Comment:
You dont need to use this long directory to get the image. Since you have the image and the file in the same folder, you can simply put the image name only.

In a better way, put all your images for this task in a folder in the html folder. So to get the image, simply use 'images/image-name'